presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -4097,13 +4097,19 @@ public void testWithHiding()
             throws Exception
     {
         assertQuery("" +
-                "WITH a AS (SELECT custkey FROM orders), " +
+                "WITH a AS (VALUES 1), " +
```

Comment:
I will extract this as separate commit. I was using this tests in regards of scope of named query. 

IMO test was wrong as result of the query was the same disregarding which named query was executed: `a` or `b::a`